CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 ## Master
 
+* Added support for local plugins
+* Added support for remote plugins
+* Added new `danger new_plugin` command to create plugins in the fastlane
 * Added new `modified_files.include?("rakelib/*_stats.rake") file globbing support - KrauseFx
+* Added printing of table summaries after running danger
```

Comment:
You left the backtick around `modified_files.include?` on the previous line open, so the following lines may show up as monospaced.